### PR TITLE
Create MOTH 2019 Article 

### PR DESCRIPTION
**Pull request by atribolet**
Pull request description
------------------------

### CODE REVIEW

File: _posts/2019-03-12-maps-on-the-hill-2019.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - Featured
+tags:
+  - maps
+  - moth
+  - ugic
+---
+
+The eighth annual Maps on the Hill (MOTH) event, hosted by the Utah Geographic Information Council (UGIC) and AGRC, was held on March 7, 2019, at the Utah State Capitol to showcase the work of the local GIS community and build awareness of GIS-related issues in Utah.
+The MOTH event consists of GIS professionals creating maps and presentations to tell a story using GIS. Participants set up easels, large maps, and monitors in the capitol rotunda to inform viewers about the participants’ chosen subject matter. 
+
```
Comment (steveoh) [inline _posts/2019-03-12-maps-on-the-hill-2019.md]: please remove extra newline

**Comment by joshgroeneveld** [comment]
I've created a photos folder in the UGIC Dropbox account that you can link to: https://www.dropbox.com/sh/ks9iwsbrkb2y38w/AADD_u2UUcFSV2GZ90BYhma4a?dl=0

**Comment by steveoh** [comment]
Thanks @joshgroeneveld I added that link to https://gis.utah.gov/about/maps-on-the-hill/. Do you want any of the photos that AGRC employees took?

**Comment by joshgroeneveld** [comment]
Sure! I've got the photos that @atribolet shared, which included photos from Nathan.  If there are more, I'd be happy to add them.

**Comment by steveoh** [comment]
Of if you have anna's then we are probably good. There might be more from sean? in our facebook album. https://www.facebook.com/pg/UtahAGRC/photos/?tab=album&album_id=2309293955758677 Feel free to add any of those.

**Comment by joshgroeneveld** [comment]
Yes...there were a dozen or so photos in the facebook album that weren't in the Dropbox folder before.  Everything should be included now.

**Comment by steveoh** [comment]
Alright, waiting on Anna to do whatever it is she decides and undraft this. Thanks Josh.

**Comment by atribolet** [comment]
Hey, @steveoh, this is ready to be posted :) Thanks!

**Pull request merged**